File: Lab5-TabBar&Container/Lab5-TabBar&Container/Lab5-TabBar&Container/Scrolling/ScrollingVC.m
//
//  ScrollingVC.m
//  Lab5-TabBar&Container
//
//  Created by 張力元 on 2019/2/21.
//  Copyright © 2019 張力元. All rights reserved.
//

#import "ScrollingVC.h"

@interface ScrollingVC ()
@property (strong, nonatomic) IBOutlet UIPageControl *pageControl;
@property (strong, nonatomic) IBOutlet UIScrollView *scrollView;

@end

@implementation ScrollingVC

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (void)scrollViewDidScroll:(UIScrollView *)sender
{
    //滑動Scroll View會呼叫此方法
    //使用Scroll View左上角的X位置/Scroll View的寬度取得目前在第幾頁
    int page = sender.contentOffset.x/sender.frame.size.width;
    //設定Page Control的頁數顯示
    _pageControl.currentPage = page;
}
@end

File: Swift/lab5/lab5_exercize/lab5_exercize/scrolling/scrollingVC.swift
//
//  scrollingVC.swift
//  lab5_exercize
//
//  Created by taizhou on 2019/7/29.
//  Copyright © 2019 taizhou. All rights reserved.
//

import UIKit

class scrollingVC: UIViewController {
    @IBOutlet var pageControl: UIPageControl!
    @IBOutlet var scrollView: UIScrollView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }
}

extension scrollingVC :UIScrollViewDelegate{
    func scrollViewDidScroll(_ scrollView: UIScrollView){
        let page = scrollView.contentOffset.x/scrollView.frame.size.width
        pageControl.currentPage = Int(page)
    }
}
